spec/openapi3.yaml

```diff
@@ -2969,6 +3203,83 @@ components:
       allOf:
         - $ref: "#/components/schemas/EmbeddedTransactionDTO"
         - $ref: "#/components/schemas/TransferTransactionBodyDTO"
+
+    # Schemas - Metadata
+    MetadataEntriesDTO:
+      type: object
+      required:
+        - metadataEntries
+      properties:
+        metadataEntries:
+          type: array
+          description: Array of account metadata.
+          items:
+            $ref: "#/components/schemas/KeyMetadataEntryDTO"
+    KeyMetadataEntryDTO:
+      type: object
+      required:
+      - key
+      - metadata
+      properties:
+        key:
+          type: integer
+          format: int64
+        metadata:
+          type: array
+          description: Array of metadata entry objects.
+          items:
+            $ref: "#/components/schemas/MetadataDTO"
+    MetadataDTO:
+      type: object
+      required:
+        - meta
+        - metadataEntry
+      properties:
+        meta:
+          $ref: "#/components/schemas/MetadataMetaDTO"
+        metadataEntry:
+          $ref: "#/components/schemas/MetadataEntryDTO"
+    MetadataMetaDTO:
+      type: object
+      required:
+        - id
+      properties:
+        id:
+          type: string
+    MetadataEntryDTO:
+      type: object
+      required:
+        - compositeHash
+        - senderPublicKey
+        - targetPublicKey
+        - scopedMetadataKey
+        - targetId
+        - metadataType
+        - valueSize
+        - value
+      properties:
+        compositeHash:
+          $ref: "#/components/schemas/Hash256"
+        senderPublicKey:
+          $ref: "#/components/schemas/PublicKey"
+        targetPublicKey:
+          $ref: "#/components/schemas/PublicKey"
+        scopedMetadataKey:
+          type: integer
```

Comment:
I propose to change this to:

- type: string 
- format: hex

to match:
> metadataKey:
>       name: key
>       in: path
>       description: Metadata key.
>       required: true
>       schema:
>         type: string
>         format: hex

@Vektrat @rg911 opinions?

Reply:
`scopedMetadataKey` is `uint64` in buffer schema, so should be string (hex) type
@dgarcia360 👍

Reply:
`Uint64` has been removed recently from the doc since the `string` format introduced.

Reply:
yes my fault, it was uint64 because i think i did this before the major uint64 refactor, will change to string-hex